src/common/options/osd.yaml.in

```diff
@@ -287,7 +287,7 @@ options:
   type: float
   level: advanced
   desc: Scrub each PG no more often than this interval
-  fmt_desc: The minimal interval in seconds for scrubbing the Ceph OSD Daemon
+  fmt_desc: The desired interval in seconds for scrubbing the Ceph OSD Daemon
```

Comment:
How about:

"The desired interval in seconds between scrubs of a specific PG."

Reply:
Made the change to the 'description'